controllers/users.js

```diff
@@ -4,6 +4,23 @@ const bcrypt = require("bcrypt")
 const User = require("../models/user")
 SALT_LENGTH = 12
 
+router.get("/:userId", async (req, res) => {
+	try {
+		const userProfile = await User.findById(req.params.userId)
```

Comment:
I think this will need to be changed to 
`const userProfile = await User.findById(req.params.userId).populate('ownedReviews')`